core/src/message/context.rs

```diff
@@ -78,30 +78,61 @@ impl ContextSettings {
         self.sending_fee
     }
 
+    /// Setter for inner sending fee field.
+    pub fn set_sending_fee(&mut self, fee: u64) {
```

Comment:
these new `set_*` methods violate aims and expected usage of context objects. Either avoid that by instantiating contexts manually or with builder in the *ext.rs*, or move this new api to some separate `impl` expression and mark it with `#[cfg(test)]` (hope it works as I think).

Reply:
Removed the setters